src/hooks/auditoriumSections.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { useCallback, useMemo } from "react";
```

Comment:
can we split the hooks here? the file turned out to be a pretty big.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1251/commits/c6a0555e9c3a1fdfa5d8691db08d65724700bed2